.github/workflows/conan-package.yml

```
@@ -21,7 +21,7 @@ on:
             - 'CURA-*'
             - '[0-9]+.[0-9]+'
         tags:
-            - '[0-9]+.[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.[0-9]*'
```

Didn't you say you wannted to add two cases here?
```suggestion
            - '[1-9]+.[0-9]+.[0-9]*'
            - '[1-9]+.[0-9]+.[0-9]'
```